### ISSUE TITLE
v2 ANSI format shows forecast by hour, but v2 JSON format every 3 hours

### ISSUE THREAD

**Issue by jperales** [chubin/wttr.in#360]
I would like to parse the hourly weather forecast (chance of precipitation and MM) from the JSON format. For this, I am using 
```bash
wget -qO - v2.wttr.in/heidelberg?format=j1
```
However I realized that the JSON format only provides the weather every 3 hours. You can check it out using the following command:
```bash
wget -qO - v2.wttr.in/heidelberg?format=j1 | grep \"time\": | sort -u
```

> "time" values are [0, 300, 600, 900, 1200, 1500, 1800, 2100]; corresponding to "00:00 .. 21:00", respectively.

Is there any way to get the hourly forecast as it is shown in the terminal (ANSI format) by default?

Thanks!
Javier

**Comment by chubin** [comment]
@jperales Yes, it is a very good qustioon. Actually, it would be great to have the `j2` format,
that provides absolutely the same data as `v2` does, but in JSON format.
